package.json

```diff
@@ -108,6 +108,7 @@
     "glob": "^7.1.2",
     "minimatch": "^3.0.4",
     "rxjs": "^6.2.1",
+    "typescript": "^2.9.2",
```

Comment:
This is a published package, it should not compile on install. Everything is already built and included in the NPM package.